### ISSUE TITLE
0.0.17 appimage - crash during check

### ISSUE THREAD

**Issue by wkitty42** [Portree-Kid/flightgear-airports#37]
```
Crashed                                                                                             check.js:39
checkGroundnet.then.catch.result @ check.js:39
Promise.catch (async)
onmessage @ check.js:38

TypeError: Cannot read property 'length' of undefined                                               check.js:40
    at Object.polygon (/tmp/.mount_flightEtPxTb/resources/workers/node_modules/@turf/turf/turf.js:235)
    at parkingNodes.forEach.parkingNode1 (check.js:239)
    at Array.forEach (<anonymous>)
    at parkingNodes.forEach.parkingNode (check.js:231)
    at Array.forEach (<anonymous>)
    at check.js:230
    at new Promise (<anonymous>)
    at checkGroundnet (check.js:47)
    at onmessage (check.js:33)
checkGroundnet.then.catch.result @ check.js:40
Promise.catch (async)
onmessage @ check.js:38
```

**Comment by wkitty42** [comment]
this may be related with #34 and what FGA is seeing as Unknown Radius...

i'm working with KRDU and there are still some ""odd"" radius sizes being used... these parking places are appearing as circles instead of the craft outline...

**Comment by Portree-Kid** [comment]
Good catch

**Comment by Portree-Kid** [comment]
Fixed in 0.0.18
